From 4c29b332e7b5f03619481a6eab9690f96ae665f2 Mon Sep 17 00:00:00 2001
From: Kevin Wolf <kwolf@redhat.com>
Date: Fri, 22 Nov 2013 14:23:29 +0100
Subject: [PATCH 11/14] qcow2: Change default for new images to compat=1.1

RH-Author: Kevin Wolf <kwolf@redhat.com>
Message-id: <1385130210-31063-2-git-send-email-kwolf@redhat.com>
Patchwork-id: 55856
O-Subject: [RHEL-7.0 qemu-kvm PATCH 1/2] qcow2: Change default for new images to compat=1.1
Bugzilla: 1026739
RH-Acked-by: Max Reitz <mreitz@redhat.com>
RH-Acked-by: Fam Zheng <famz@redhat.com>
RH-Acked-by: Paolo Bonzini <pbonzini@redhat.com>

By the time that qemu 1.7 will be released, enough time will have passed
since qemu 1.1, which is the first version to understand version 3
images, that changing the default shouldn't hurt many people any more
and the benefits of using the new format outweigh the pain.

qemu-iotests already runs with compat=1.1 by default.

Signed-off-by: Kevin Wolf <kwolf@redhat.com>
Reviewed-by: Stefan Hajnoczi <stefanha@redhat.com>
Reviewed-by: Eric Blake <eblake@redhat.com>
Signed-off-by: Stefan Hajnoczi <stefanha@redhat.com>
(cherry picked from commit 8ad1898cf1f5314731123afce057e5cf74fd2f01)

Conflicts:
	block/qcow2.c

Signed-off-by: Kevin Wolf <kwolf@redhat.com>
---
 block/qcow2.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Signed-off-by: Miroslav Rezanina <mrezanin@redhat.com>
---
 block/qcow2.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/block/qcow2.c b/block/qcow2.c
index f852d64..897e48a 100644
--- a/block/qcow2.c
+++ b/block/qcow2.c
@@ -1598,7 +1598,7 @@ static int qcow2_create(const char *filename, QEMUOptionParameter *options,
     int flags = 0;
     size_t cluster_size = DEFAULT_CLUSTER_SIZE;
     int prealloc = 0;
-    int version = 2;
+    int version = 3;
     Error *local_err = NULL;
     int ret;
 
-- 
1.7.1

